doc/src/manual/profile.md

```diff
@@ -37,17 +37,17 @@ any alternatives.
 Let's work with a simple test case:
 
 ```julia
-function myfunc()
-    A = rand(100, 100, 200)
-    maximum(A)
-end
+julia> function myfunc()
+           A = rand(100, 100, 200)
+           maximum(A)
+       end
 ```
 
 It's a good idea to first run the code you intend to profile at least once (unless you want to
 profile Julia's JIT-compiler):
 
 ```julia
-julia> myfunc()  # run once to force compilation
+julia> myfunc() # run once to force compilation
 ```
 
 Now we're ready to profile this function:
```

Comment:
I can not reproduce the trace in the hidden block just below:
```julia
julia> Profile.clear(); @profile myfunc(); Profile.print()
13 ./event.jl:73; (::Base.REPL.##1#2{Base.REPL.REPLBackend})([2...
 13 ./REPL.jl:97; macro expansion
  13 ./REPL.jl:66; eval_user_input(::Any, ::Base.REPL...
   13 ./boot.jl:235; eval(::Module, ::Any)
    13 ./<missing>:?; anonymous
     13 ./profile.jl:23; macro expansion
      6 ./REPL[1]:2; myfunc()
       6 ./random.jl:431; rand!(::MersenneTwister, ::Arra...
        6 ./dSFMT.jl:84; dsfmt_fill_array_close_open!(::...
      7 ./REPL[1]:3; myfunc()
       7 ./reduce.jl:270; _mapreduce(::Base.#identity, ::...
        3 ./reduce.jl:420; mapreduce_impl(::Base.#identity...
        1 ./reduce.jl:426; mapreduce_impl(::Base.#identity...
        3 ./reduce.jl:428; mapreduce_impl(::Base.#identity...
```

I left it as is, since much of the text in the sections below refers to this, and in particular to the lines not shown in my case. (`client.jl` etc)